Source/SwiftLintBuiltInRules/Rules/Lint/UnneededSynthesizedInitializerRuleExamples.swift

```diff
@@ -0,0 +1,426 @@
+                    var bar: String
+
+                   ↓init(bar: String) {
+                        self.bar = bar
+                    }
+                }
+                """),
+        Example("""
+                private struct Foo {
+                    let bar: String
+
+                   ↓init(bar: String) {
+                        self.bar = bar
+                    }
+                }
+                """),
+        Example("""
+                fileprivate struct Foo {
+                    let bar: String
+
+                   ↓init(bar: String) {
+                        self.bar = bar
+                    }
+                }
+                """),
+        Example("""
+                internal struct Foo {
+                    fileprivate var bar: String
+
+                   ↓fileprivate init(bar: String) {
+                        self.bar = bar
+                    }
+                }
+                """),
+        Example("""
+                internal struct Foo {
+                    private var bar: String
+
+                   ↓private init(bar: String) {
+                        self.bar = bar
+                    }
+                }
+                """),
+        Example("""
+                struct Foo {
+                    var foo: String
+                    var bar: String
+
+                   ↓init(foo: String, bar: String) {
+                        self.foo = foo
+                        self.bar = bar
+                    }
+                }
+                """),
+        Example("""
+                internal struct Foo {
+                    var bar: String
+
+                   ↓internal init(bar: String) {
+                        self.bar = bar
+                    }
+                }
+                """),
+        Example("""
+                struct Foo {
+                    var bar: String = ""
+
+                   ↓init() {
+                        // Empty initializer will be generated automatically
+                        // when all vars have default values.
+                    }
+                }
+                """),
+        Example("""
+                struct Foo {
+                    var bar: String = ""
+
+                   ↓init() {
+                        // Empty initializer
+                    }
+
+                   ↓init(bar: String = "") {
+                        self.bar = bar
+                    }
+                }
+                """),
+        Example("""
+                struct Foo {
+                    var bar = ""
+
+                   ↓init(bar: String = "") {
+                        self.bar = bar
+                    }
+                }
+                """)
+    ]
+
+    static let corrections = [
+        Example("""
+                struct Foo {
+                    let bar: String
+
+                   ↓init(bar: String) {
+                        self.bar = bar
+                    }
+
+                    func baz() {
```

Comment:
Could you add a comment above the method declaration to proof that it is not removed by the rewriter?